tests/references/__snapshots__/test_api.ambr

```diff
@@ -616,297 +616,6 @@
     'version': 0,
   })
 # ---
-# name: TestCreateOTU.test[True--True][db]
-  list([
-    dict({
-      '_id': 'bf1b993c',
-      'abbreviation': '',
-      'isolates': list([
-      ]),
-      'last_indexed_version': None,
```

Comment:
What's going on here? Seem unrelated to the PR

Reply:
I guess I accidentally updated this snapshot, I have undone this change.